src/parsing/ris_live.go

```diff
@@ -0,0 +1,46 @@
+package parsing
+
+import (
+	"BGPAlert/common"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/gorilla/websocket"
+)
+
+func main() {
+	// RIS Live websocket url
+	socketUrl := "ws://ris-live.ripe.net/v1/ws/"
+
+	// create websocket connection to ris live websocket
+	conn, _, err := websocket.DefaultDialer.Dial(socketUrl, nil)
+	if err != nil {
+		log.Fatal("Websocket connection error:", err)
+	}
+	defer conn.Close()
+
+	for {
```

Comment:
I don't remember my Go but I think your code should somehow wait on the messages it's reading using a `select` statement or something like that. I might be completely talking nonsense but @apai-fsly, can you please help us with this?

Reply:
select is specifically for channels in go.